examples/bench/README.md

```diff
@@ -0,0 +1,59 @@
+
+# `tsh` Bench
+
+
+## Using `tsh` bench 
+
+Specify the rps and duration with `--rate` and `--duration`. By default rate is 10 requests per seconds and duration is 1s. You will need to provide `[user@]host` and a command to run. 
+
+Each request will be run concurrently in its own goroutine. 
+
+```
+# generate requests at 100 requests per second
+# for 30 seconds
+tsh bench --rate=100 --duration=30s localhost ls -l /
+```
+
+
+## Exporting Latency Profile 
+
+Run `tsh bench` as usual, but use the `--export` flag to export. Use `--path` to specify a directory to save the file to. If the path is not specified, the file will be saved to your current working directory. 
+
+Example:  
+
+`tsh bench --export --rate 100 --duration 10s localhost ls -l /` 
+
+The file its saved in the following format: latency_profile_2006-10-27_15:04:05.txt
```

Comment:
If you update the logic to export both service & response time series, make sure to update the instructions here to explain which is which & how to plot both.